doc/specs/#4999 - Improved keyboard handling in Conpty.md

```diff
@@ -0,0 +1,514 @@
+---
+author: Mike Griese @zadjii-msft

```

Comment:
Okay yes, this branch branched off the spec branch. So once that gets merged in, this will go away on its own.